tls/test/test_hello_message.py

```diff
@@ -219,6 +230,30 @@ def test_hello_from_bytes_with_unsupported_extension(self):
                 client_hello_packet
             )
 
+    def test_parse_client_certificate_url_extension(self):
+        """
+        :py:func:`tls.hello_message.ClientHello` parses a packet with
+        CLIENT_CERTIFICATE_URL extension.
+        """
+        record = ClientHello.from_bytes(
+            self.client_hello_packet_with_client_certificate_url_extension
+        )
+        assert len(record.extensions) == 1
+        assert (record.extensions[0].type ==
+                enums.ExtensionType.CLIENT_CERTIFICATE_URL)
+        assert record.extensions[0].data == {}
```

Comment:
I'd prefer this be 

```python
assert record.extensions[0].data == Container()
```

Since `data` is actually a `Container` and I prefer to forget that `Container` subclasses `dict`.